src/dcqc/updaters.py

```diff
@@ -0,0 +1,56 @@
+from csv import DictWriter
+from pathlib import Path
+from typing import Dict, List
+
+from dcqc.parsers import CsvParser
+from dcqc.suites.suite_abc import SuiteABC
+
+
+class CsvUpdater:
+    input_path: Path
+    output_path: Path
+    parser: CsvParser
+
+    def __init__(self, input_path: Path, output_path: Path):
+        self.output_path = output_path
+        self.parser = CsvParser(input_path)
+
+    def update(self, suites: List[SuiteABC]):
+        suite_dict: Dict[
```

Comment:
What was the error that mypy gave ?

Look into collections.defaultdict.  that might be helpful and will simplify your logic below.

Reply:
the error was `Need type annotation for "suite_dict" (hint: "suite_dict: Dict[<type>, <type>] = ...")  [var-annotated]`. 

I will look into defaultdict

Reply:
I added defaultdict to the logic. The error is gone and the logic is simpler. Thanks!